docs/docs/troubleshooting/hardware-issues.mdx

```diff
@@ -0,0 +1,183 @@
+
+</TabItem>
+<TabItem value="multimeter">
+:::tip
+If you are unfamiliar with the use of a multimeter, [Sparkfun's guide](https://learn.sparkfun.com/tutorials/how-to-use-a-multimeter) may prove a helpful resource.
+:::
+
+A multimeter is the most reliable tool for troubleshooting hardware issues. Here is a helpful checklist for you to use while probing your device with the multimeter:
+
+- Are any of the pins shorted that shouldn't be shorted?
+- Are all the diodes/components working and in the correct orientation?
+- Does pressing a keyswitch close the affected circuit?
+- When the device is plugged in, are all the pins at the expected voltage? (Typically ground or 3V3, which is where depends on the design and firmware)
+- Are there any failed solder joints?
+
+:::tip
+To check a solder joint, it is best to probe the two points either side of it if you can. For example:
+![A section of a schematic with a particular joint marked in green](../assets/troubleshooting/multimeter/schematic_probe_example.png)
+To check if the joint marked in red is faulty, you would first check that the diode is working, and then test whether the joint is faulty by probing the two purple joints. If you get the same results as probing the diode directly, then there is no problem with the joint.
+:::
+
+</TabItem>
+<TabItem value="visual">
+Visual inspection is generally less reliable, but a lot easier than either of the other approaches. Generally, you want to be looking for solder bridges, exposed copper, broken PCB traces, lifted pads, and similar artifacts.
+
+For through-hole soldering specifically, the below infographic can be quite helpful.
+![A helpful soldering infographic. Source: Sparkfun](https://cdn.sparkfun.com/assets/c/d/a/a/9/523b1189757b7fb36e8b456b.jpg)
+:::tip
+The above image was taken from [Sparkfun's guide on soldering through-hole components](https://learn.sparkfun.com/tutorials/how-to-solder-through-hole-soldering), which may be of use if you are less experienced with soldering.
+:::
+
+</TabItem>
+</PinTabs>
+
+### Resolving Issues
+
+Many issues can be resolved by reworking the affected solder joints (Look up a tutorial on how to do so for your component). Sometimes a pin ends up broken though, in which case you will need to "bodge" (solder) a wire from a spare GPIO pin to the connection in need of a working pin.
+
+Once you have done so, you will need to adjust the `kscan` of your keyboard slightly. The recommended approach to doing so is via your `zmk-config`:
+
+1. Search through the files defining your board/shield for the definition of the keyboard's `kscan`. This is typically a`.dts`, `.dtsi`, or an `.overlay` file.
+2. Make a file called `<my_keyboard>.overlay` file in your `zmk-config`, where `my_keyboard` is the name of the affected keyboard (part).
```

Comment:
I believe this won't work for split shields right now, due to #1382.

Reply:
Turned it into modifying `.keymap` instead.

Reply:
I think here we need some sort of clarification that .keymap applies to all parts of a split keyboard, so all parts would need the same bodge. For splits, do we have a choice other than documenting the per-part keymap workaround?